drivers/axi_core/spi_engine/spi_engine.h

```diff
@@ -144,6 +144,8 @@ struct spi_engine_offload_init_param {
 	uint32_t	rx_dma_baseaddr;
 	/** Base address where the TX DMAC core is situated */
 	uint32_t	tx_dma_baseaddr;
+	/** DMAC flags */
```

Comment:
A comment will be helpful.
/* If DMA flags is 0, the default value DMA_CYCLIC will be used */

Reply:
Sure.